packages/project/src/index.ts

```diff
@@ -108,3 +108,18 @@ export function defaultConfig(configFilePath: string): ProjectOptions {
     launch: []
   }
 };
+
+export class AssertionError extends Error {
+  constructor(message: string) {
+    super(message);
+
+    Object.setPrototypeOf(this, new.target.prototype);
+  }
+}
+
+export function assert(condition: unknown, msg?: string): asserts condition {
+  if (!condition) {
+    throw new AssertionError(msg || 'assertion error');
+  }
+}
```

Comment:
I'm not sure why `assert` helper goes here? `@bigtest/project` is for working doing things like finding the bigtest config, etc... It looks like this also isn't being used anywhere? We may need a `@bigtest/assert` package, or just use [assert](https://www.npmjs.com/package/assert) from npm?

Reply:
this does more than assert, check out the `: asserts condition {` on line 120

[This explains it](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions)

Reply:
Oh, ok. Do we need it in a shared location? `@bigtest/project` isn't really the place for that. It's more for project configuration (directory, testfiles, etc...)

Reply:
To be honest, if it is shared, I don't know the right place for it. Maybe we create a `@bigtest/lib` package?